modules/asset_import/config.py

```diff
@@ -0,0 +1,5 @@
+def can_build(env, platform):
+    return (env['tools'] and platform not in ["android", "ios"])
```

Comment:
We don't build tools on Android and iOS for now, so the second part is redundant. When we do attempt to make the editor to build on Android, there will be a lot more work involved anyway, so no need to future proof it IMO.

Reply:
done.